test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -287,6 +289,23 @@ void test_ucp_peer_failure::do_test(size_t msg_size, int pre_msg_count,
 
     EXPECT_EQ(UCS_OK, m_err_status);
 
+#if HAVE_IB
```

Comment:
maybe extract this code to helper function: `void set_ud_trimeout()`

Reply:
done
this is extracted to `void ucp_test_base::entity::set_ib_ud_timeout(double timeout_sec)`